types/scene/wlr_scene.c

```diff
@@ -2042,6 +2176,7 @@ static void scene_entry_render(struct render_list_entry *entry, const struct ren
 		enum wl_output_transform transform =
 			wlr_output_transform_invert(scene_buffer->transform);
 		transform = wlr_output_transform_compose(transform, data->transform);
+		enum corner_location buffer_corners = scene_buffer->corners;
```

Comment:
Nit: changes like these probably can best put in their own PR/commit (depending on if you're planning to squash and rebase or just, rebase, the latter is easiest)